newIDE/app/src/Utils/GDevelopServices/Extension.js

```diff
@@ -5,6 +5,7 @@ import semverSatisfies from 'semver/functions/satisfies';
 
 export type ExtensionShortHeader = {|
   shortDescription: string,
+  authors?: Array<string>,
```

Comment:
(to be sure it's not forgot: new type here to be imported from User.js presumably (we keep types defined in the same API as they are in the backend).